yarn.lock

```diff
@@ -117,6 +117,20 @@
     resolve "^1.14.2"
```

Comment:
@DianaVargasOrellana, I think this file should not be uploaded to the main repository.